substrate/frame/support/src/traits/tasks.rs

```diff
@@ -0,0 +1,87 @@
+// This file is part of Substrate.
+
+// Copyright (C) Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! Contains the [`Task`] trait, which defines a general-purpose way for defining and executing
+//! service work, and supporting types.
+
+use codec::FullCodec;
+use scale_info::TypeInfo;
+use sp_runtime::DispatchError;
+use sp_std::{fmt::Debug, iter::Iterator, vec, vec::IntoIter};
+use sp_weights::Weight;
```

Comment:
Why are they still exported here?

Reply:
They are being used in the trait definition below (not being exported). The export is separate via `__private`.